SwissTransferFeatures/SettingsView/EditSettings/Cells/EditSettingsCell.swift

```diff
@@ -0,0 +1,56 @@
+/*
+ Infomaniak SwissTransfer - iOS App
+ Copyright (C) 2024 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import InfomaniakCoreSwiftUI
+import STResources
+import SwiftUI
+
+struct EditSettingsView: View {
+    var leftIconAsset: STResourcesImages?
+    let selected: Bool
+    let label: String
+    let action: () -> Void
+
+    var body: some View {
+        Button(action: action) {
+            HStack(spacing: IKPadding.small) {
+                if let leftIconAsset {
+                    Image(asset: leftIconAsset)
+                        .iconSize(.large)
```

Comment:
suggestion: I would not rely on the color of the icon, use icons as template, and use a foreground color

Reply:
Ok, this is why you ask about a change on the `template-rendering-intent` next.

Reply:
Changed that for all assets used by the Settings view